docs/features/containers.md

```diff
@@ -357,6 +357,16 @@ const container = await new GenericContainer("alpine").start();
 await container.restart();
 ```
 
+## Committing a container to an image
+
+```javascript
+const container = await new GenericContainer("alpine").start();
+// Do something with the container
+await container.exec(["sh", "-c", `echo 'hello world' > /hello-world.txt`]);
+// Commit the container to an image
+const newImageId = await container.commit({ repo: "my-repo", tag: "my-tag", deleteOnExit: false });
```

Comment:
Could we omit the default and add a line explaining that by default the image will be cleaned up, and give a second example overriding this behaviour?

Could we also give an example how this `newImageId` is used? I think it's not clear what the value is. If we add like `const containerFromCommit = await new GenericContainer(newImageId).start();` I think that'll be nice and clear